liquibase-standard/src/main/java/liquibase/Liquibase.java

```diff
@@ -1227,26 +1227,17 @@ public void clearCheckSums() throws LiquibaseException {
      * @deprecated Use {link {@link CommandScope(String)}.
      */
     @Deprecated
-    public final CheckSum calculateCheckSum(final String changeSetIdentifier) throws LiquibaseException {
+    public final CheckSum calculateCheckSum(final String changeSetPath, final String changeSetId, final String changeSetAuthor) throws LiquibaseException {
```

Comment:
Even though this is a deprecated method it is a public API method and there might be people still using it, so I don't think it's a good idea to change its signature. I'd suggest to leave it as it was previously.